panda/src/putil/loaderOptions.I

```diff
@@ -55,6 +58,44 @@ get_texture_flags() const {
   return _texture_flags;
 }
 
+/**
+ * Remember all the options for the textures being loaded.
+ * These are used to gurantee that unique textures are not ignored. 
+*/
+
+INLINE void LoaderOptions::
+set_texture_format(int format){
+  _texture_format= format;
+}
+
+/**
+ *  
+ */
+INLINE int LoaderOptions::
+get_texture_format() const {
+  return _texture_format;
+}
+
+INLINE void LoaderOptions::
+set_texture_compress(int compress){
+  _texture_compress = compress;
+}
+
+INLINE int LoaderOptions::
+get_texture_compress() const {
+  return _texture_compress;
+}
+
+INLINE void LoaderOptions::
+set_texture_quality(int quality){
+  _texture_quality = quality;
+}
+
+INLINE int LoaderOptions::
+get_texture_quality() const {
+  return _texture_quality;
+}
```

Comment:
The `/**` blocks are used to generate the API reference, so there should ideally be one for every method, with an appropriate message (though getters/setters are usually pretty obvious, so it's not that pressing)

Reply:
This is trivial to add so I did it.